lib/philomena_web/forum_json.ex

```diff
@@ -0,0 +1,24 @@
+defmodule PhilomenaWeb.ForumJson do
+
+  def as_json(%{access_level: "normal"} = forum) do
+    %{
+      id: forum.id,
+      name: forum.name,
+      short_name: forum.short_name,
+      description: forum.description,
+      topic_count: forum.topic_count,
+      post_count: forum.post_count
+    }
+  end
```

Comment:
Style: function clauses of the same name and arity should have no blank line between them